### ISSUE TITLE
update ontology

### ISSUE THREAD

**Issue by tillmo** [tillmo/DOL#269]
There have been some changes in chapter 4. I have added the term "infrastructure axiom". Also, some of the definitions of terms have changes. Please change the ontology accordingly. The convenience document at https://github.com/tillmo/DOL/raw/master/Standard/dol-diff.pdf can help.

**Comment by clange** [comment]
I am done (based on dol-diff.pdf).

One important question remains:

In the terminology section of the standard we have a term "simple theoroidal logic translation", which didn't exist in the ontology, so I've added it.

However, we did have in the ontology the class `SimpleTheoroidalMapping`, which has no counterpart in the terminology.

Do we still need the latter class?

"Simple theoroidal logic translation" is more specific than "simple theoroidal mapping" in two ways:
 1. It's a translation (not a reduction).
 1. It's a _logic_ translation (not, e.g., a language translation).

We could express the same as the intersection of orthogonal classes: `SimpleTheoroidalMapping and Translation and LogicTranslation`. Having the specific class `SimpleTheoroidalLogicTranslation` in the ontology makes sense if there if there is no simple theoroidal reduction and no simple theoroidal language translation. And even then I'd suggest to keep the class `SimpleTheoroidalMapping`, as keeping it will enable us to define `SimpleTheoroidalLogicTranslation` in a formally more precise way.

tl;dr: I added SimpleTheoroidalLogicTranslation but kept SimpleTheoroidalMapping.

**Comment by tillmo** [comment]
good, many thanks!
